docs/metrics/getting-started-prometheus-grafana/getting-started-prometheus-grafana.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
```

Comment:
please make it match https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/docs/metrics/getting-started/getting-started.csproj

Reply:
updated